caom2utils/caom2utils/fits2caom2.py

```diff
@@ -2406,6 +2559,44 @@ def _get_from_list(self, lookup, index, current=None):
         self.logger.debug('{}: value is {}'.format(lookup, value))
         return value
 
+    def _get_from_table(self, lookup, extension):
+        """
+        Return a space-delimited list of all the row values from a column.
+
+        This is a straight FITS BINTABLE lookup. There is no support for
+        default values. Unless someone provides a compelling use case.
+
+        :param lookup: where to find the column name
+        :param extension: which extension
+        :return: A string, which is a space-delimited list of all the values.
+        """
+        value = ''
+        try:
+            keywords = self.blueprint._get(lookup, extension)
+        except KeyError:
+            self.add_error(lookup, sys.exc_info()[1])
+            self.logger.debug(
+                'Could not find {!r} in fits2caom2 configuration.'.format(
+                    lookup))
+            return value  # TODO what to return when failing lookup
```

Comment:
I think it should throw the KeyError exception and let the caller deal with it.

Reply:
Done.